tests/e2e/specs/api/BuildPushRunPodmanContainerAPI.spec.ts

```diff
@@ -81,6 +72,10 @@ fi
 oc logs test-hello-pod
 `;
 
+	const factoryUrl: string = BASE_TEST_CONSTANTS.IS_CLUSTER_DISCONNECTED()
```

Comment:
I would suggest taking the `TS_SELENIUM_FACTORY_GIT_REPO_URL` value into account and adding an airgap-specific constant: `TS_SELENIUM_AIRGAP_FACTORY_GIT_REPO_URL`.

This will allow testing Dev Spaces on **non-x86_64** infrastructure.